app/views/idv/in_person/state_id.html.erb

```diff
@@ -1,13 +1,67 @@
 <% title t('titles.doc_auth.verify') %>
 
-<h1 class="h3 margin-y-0">
-  <%= t('in_person_proofing.headings.state_id') %>
-</h1>
+
+<%= render PageHeadingComponent.new.with_content(t('in_person_proofing.headings.state_id')) %>
+
+<p>
+  <%= t('in_person_proofing.body.state_id.info_html') %>
+</p>
 
 <%= validated_form_for :doc_auth,
                        url: url_for,
                        method: 'put',
                        html: { autocomplete: 'off', class: 'margin-y-5' } do |f| %>
+
+ <div class="margin-bottom-4">
+   <%= f.input :first_name,
+               label: t('in_person_proofing.form.state_id.first_name'),
+               wrapper: false, required: true, maxlength: 255,
+               input_html: { aria: { invalid: false } } %>
+   <span class='usa-error-message margin-top-1 display-if-invalid display-if-invalid--value-missing margin-bottom-1' role='alert'>
+     <%= t('simple_form.required.text') %>
+   </span>
+ </div>
+
+  <div class="margin-bottom-4">
+    <%= f.input :last_name,
+                label: t('in_person_proofing.form.state_id.last_name'),
+                wrapper: false, required: true, maxlength: 255,
+                input_html: { aria: { invalid: false } } %>
+    <span class='usa-error-message margin-top-1 display-if-invalid display-if-invalid--value-missing margin-bottom-1' role='alert'>
+      <%= t('simple_form.required.text') %>
+    </span>
+  </div>
+
+  <div class="margin-bottom-4">
+    <%= f.input :dob,
+                as: :date, html5: true, label: t('in_person_proofing.form.state_id.dob'),
+                hint: t('in_person_proofing.form.state_id.dob_hint'), wrapper: false,
+                required: true, maxlength: 255,
+                input_html: { aria: { invalid: false } } %>
+    <span class='usa-error-message margin-top-1 display-if-invalid display-if-invalid--value-missing margin-bottom-1' role='alert'>
+      <%= t('simple_form.required.text') %>
+    </span>
+  </div>
+
+  <div class="margin-bottom-4">
+    <%= f.input :state_id_jurisdiction,
+                collection: us_states_territories,
+                label: t('in_person_proofing.form.state_id.state_id_jurisdiction'), required: true,
+                hint: t('in_person_proofing.form.state_id.state_id_jurisdiction_hint'),
+                prompt: t('in_person_proofing.form.state_id.state_id_jurisdiction_prompt') %>
+  </div>
+
+  <div class="tablet:grid-col-7 margin-bottom-4">
+    <%= f.input :state_id_number,
+                label: t('in_person_proofing.form.state_id.state_id_number'),
+                hint: t('in_person_proofing.form.state_id.state_id_number_hint'),
+                wrapper: false, required: true, maxlength: 255,
+                input_html: { aria: { invalid: false } } %>
+    <span class='usa-error-message margin-top-1 display-if-invalid display-if-invalid--value-missing margin-bottom-1' role='alert'>
+      <%= t('simple_form.required.text') %>
+    </span>
```

Comment:
Can we try to use `ValidatedFieldComponent` here? I've been trying gradually phase out and remove the `display-if-invalid` stuff as legacy. I think you could probably get the same effect by:

```suggestion
    <%= render ValidatedFieldComponent.new(
                 form: f,
                 name: :state_id_number,
                 label: t('in_person_proofing.form.state_id.state_id_number'),
                 hint: t('in_person_proofing.form.state_id.state_id_number_hint'),
                 wrapper: false,
                 required: true,
                 maxlength: 255,
              ) %>
```

Reply:
I changed them all to ValidatedFieldComponents in 13172ff9c8accbd8384d864ecaaff095c6275bff